extranet/src/SignupForm.tsx

```diff
@@ -81,6 +117,10 @@ const SignupForm: React.FC = () => {
       />
     </div>
     <div className={classes.formContainer}>
```

Comment:
an example based on the code suggestion above
```suggestion
    <div className={formContainer}>
```
same for the rest of code

Reply:
it's done